src/main/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/SplitNCigarReads.java

```diff
@@ -192,7 +192,7 @@ public void closeTool() {
         if (outputWriter != null ) { outputWriter.close(); }
         try {if (referenceReader != null) { referenceReader.close(); } }
         catch (IOException ex) {
-            throw new UserException.MissingReference("Could not find reference file");
+            throw new UserException.MissingReference("Could not find reference file", true);
```

Comment:
This error message should end with a period. Might be worth defining a static final String variable for it too, since it's used in multiple places.

Reply:
done